presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -775,6 +778,11 @@ std::string PrestoServer::getLocalIp() const {
       "Could not infer Node IP. Please specify node.ip in the node.properties file.");
 }
 
+std::string PrestoServer::getBaseSpillPath() const {
+  const auto baseSpillPath = SystemConfig::instance()->spillerSpillPath();
+  return baseSpillPath.has_value() ? baseSpillPath.value() : "";
+}
+
```

Comment:
It looks we want a default value for this config. Then https://github.com/prestodb/presto/blob/e6ae941ce7a85b42cec377f5dcc62161966605d1/presto-native-execution/presto_cpp/main/common/Configs.cpp#L244 is usually the place we specify a default value.